### DISCUSSION TITLE
How to show certain attributes only based on selection in the other attribute

### DISCUSSION THREAD

**Discussion by KasthuriAnandhan** [pimcore/pimcore#16656] [Q&A / Support]
Hi @kingjia90 

Is there any posibilities to define class within class or in the below image you can see set of attributes in object, the object contains some fields. The **Platform** attributes contain 3 values that is **BigCommerce**, **WooCommerce**, **BigCommerce & WooCommerce** if I select **BigCommerce store_url and access_token** only have to show, If I select **WooCommerce consumer_key and consumer_secret** have to show if I select **BigCommerce & Woocommerce all the  attributes store_url, access_token, consumer_key, consumer_secret, client_id, storee_hash** will have to show is there any option is there in PIMCORE for this and I know there is a possibility to do this through object bricks but that will open the  next next tabs I don't need like this my requirement is like mentioned above. Hope you can help me. and let me know if there is possibility are there  or not to do this in PIMCORE other than object bricks.
![image](https://github.com/pimcore/pimcore/assets/125338676/79cbe94e-8289-4d4d-9302-1723194743c3)

Thanks

**Comment by kingjia90** [comment]
Hi @KasthuriAnandhan,

Thank you for posting.
I am sorry but i won't be able to help you, hope somebody could step in. 




PS: not sure how it works, but just an warning/heads-up that the token is publicly visible in the screenshot 🫣

**Comment by KasthuriAnandhan** [comment]
Okay Thanks Jia. No problem that was dummy token.
